indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1854,7 +1858,7 @@ def __draw_badge(self, registrant, posx, posy):
                 if argumentType == Registrant:
```

Comment:
Well, if I had to check every line of this file, I am sure I would find a lot of dead code! Does it really make sense to remove it now?